src/ui/Card/index.tsx

```diff
@@ -24,7 +24,7 @@ const CardContainer = ({
 
 export type CardProps = {
   theme?: "light" | "dark";
-  title: {
+  title?: {
```

Comment:
Why is this optional now?

Reply:
Because on the FeedPage there is no header. In that case it should not render that part.